source/fundamentals/crud/run-command.txt

```diff
@@ -0,0 +1,159 @@
+.. _golang-run-command:
+
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a raw **database command** with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+Compose a Command
+-----------------
+
+The {+driver-short+} provides two methods to run a database command. The
+``RunCommand()`` method executes the given command against the database
+and returns a ``SingleResult`` type. The ``RunCommandCursor()`` method
+executes the given command against the database and parses the response
+as a cursor.
+
+.. note:: Read Preference
```

Comment:
Optional: Consider moving this to the end of the **Compose a Command** section. It interrupts the flow somewhat as is.

Reply:
agreed-- especially since the information about parameters is right under it.